CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Version 1.2.0
+
+* Added proof producing versions for Sign, Max, Absolute Value and Disjunction constraints
```

Comment:
I think we're still on v1.1.0 actually as it hasn't been released. Maybe the `v1.1.0` title should `Next release` to avoid confusion?